### PR TITLE
glade: Update to 3.38.2

### PR DESCRIPTION
**Pull request by xjuan**
Glade now has all the patches needed to build
Add glib-networking dependency needed for survey

### CODE REVIEW

**Comment by xjuan** [comment]
Hello guys, I just made a Glade bugfix release to include all the windows fixes.
Sorry I did not make it on time before you updated it.

If you ever find a build error please report it in https://gitlab.gnome.org/GNOME/glade/-/issues
and I will try to include it in the next point release, that way you do not have to deal with patches

thanks for all the work you do, this is a pretty cool!

File: mingw-w64-glade/PKGBUILD [hunk 1]
```diff
@@ -3,8 +3,8 @@
 _realname=glade
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=3.38.1
-pkgrel=4
+pkgver=3.38.2
+pkgrel=5
```
Comment (lazka) [inline mingw-w64-glade/PKGBUILD:7]: pkgrel=1

**Comment by lazka** [comment]
thanks!

File: mingw-w64-glade/PKGBUILD (hunk 1 above)
Comment (xjuan) [reply mingw-w64-glade/PKGBUILD:7]: ohh IC

**Pull request merged**